dot/node.go

```diff
@@ -313,47 +353,23 @@ func NewNode(cfg *Config, ks *keystore.GlobalKeystore) (*Node, error) {
 		metrics.PublishMetrics(address)
 	}
 
-	gd, err := stateSrvc.Base.LoadGenesisData()
-	if err != nil {
-		return nil, err
-	}
+	return node, nil
+}
 
+func setupTelemetry(cfg *Config, genesisData *genesis.Data) (*telemetry.Mailer, error) {
```

Comment:
Please use named returns

```suggestion
func setupTelemetry(cfg *Config, genesisData *genesis.Data) (mailer *telemetry.Mailer, err error) {
```